spec/factories/trips.rb

```diff
@@ -1,8 +1,8 @@
 FactoryBot.define do
   factory :trip do
     ticket_price { Faker::Number.between(from: 1, to: 10_000) }
-    total_booked { Faker::Number.between(from: 1, to: 60) }
     date { Faker::Date.between(from: '2023-05-01', to: '2024-05-01') }
+    total_booked { Faker::Number.between(from: 1, to: 60) }
```

Comment:
do we need to consider the available seat or seat counts on a trip?

Reply:
Bhaia, we are not saving seat count from parameter, seat count comes from the value of bus capacity.

Reply:
is there any chance that total_booked will be higher than the actual seat capacity as `Faker::Number.between(from: 1, to: 60)` can choose a number between 1 to 60?

Reply:
Thanks for pointing it out vaia. Removed `Faker::Number.between(from: 1, to: 60)` as I already have made the default value of total_booked 0 in Model file.